src/screens/OtherProfile/index.js

```diff
@@ -255,15 +264,15 @@ const OtherProfile = () => {
       if (dataMain.is_following)
         return (
           <GlobalButton onPress={() => handleSetUnFollow()}>
-            <View style={styles.buttonFollowing}>
-              <Text style={styles.textButtonFollowing}>Following</Text>
+            <View style={styles.buttonFollowing(isAnonimity)}>
+              <Text style={styles.textButtonFollowing(isAnonimity)}>Following</Text>
             </View>
           </GlobalButton>
         );
 
       return (
         <GlobalButton onPress={() => handleSetFollow()}>
-          <View style={styles.buttonFollow}>
+          <View style={styles.buttonFollow(isAnonimity)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this hunk also seem to be related to the introduction of a new color scheme based on user anonymity. As mentioned before, please ensure these color changes do not negatively affect the accessibility of the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->